app/components/member/Edit/remove.js

```diff
@@ -41,32 +41,25 @@ class Remove extends React.PureComponent {
       return null;
     }
 
-    const loading = this.state.removing && (
-      this.props.isSelf
-        ? 'Leaving Organization…'
-        : 'Removing from Organization…'
-    );
+    const loading = this.state.removing && 'Removing from Organization…';
 
     return (
-      <Button
-        theme="default"
-        outline={true}
-        loading={loading}
-        onClick={this.handleRemoveClick}
-      >
-        {
-          this.props.isSelf
-            ? 'Leave Organization'
-            : 'Remove from Organization'
-        }
-      </Button>
+      <Panel className="mt3">
+        <Panel.Header>Remove from Organization</Panel.Header>
+        <Panel.Section>
+          <p>Removing this user will immediately revoke their access to this organization.</p>
+        </Panel.Section>
+        <Panel.Section>
+          <Button theme="default" outline={true} loading={loading} onClick={this.handleRemoveClick}>
+            Remove from Organization
+          </Button>
+        </Panel.Section>
+      </Panel>
     );
   }
 
   handleRemoveClick = () => {
-    const message = this.props.isSelf
-      ? 'Removing yourself will immediately revoke your access to this organization.'
-      : 'Removing this user will immediately revoke their access to this organization.';
+    const message = 'Removing this user will immediately revoke their access to this organization.';
```

Comment:
As I was going through the pages of all the users, it'd say one thing, but then when you look at yourself, all the words changed, and my brain had to re-parse what the button was saying. My feels were we didn't gain much for the overhead - and it's probably to be consistent (also makes for easier unit testing since it's the one button we test for, not 2)